help/md/median.md

```diff
@@ -1,9 +1,23 @@
 ## name
 median
 ## title
+Median of a set of numbers
 ## description
+Finds the median of a sorted vector of numbers.
 ## details
+The vector is sorted when `median` is used finding the
```

Comment:
I'm having trouble parsing the 1st sentence -- are there missing or extra words? In the 2nd sentence, it should say "no such value" instead of "no two such values"